### PR TITLE
Update qualys-vmdr-source.md - Provide guidance on forwarding Qualys …

### PR DESCRIPTION
**Pull request by mvirga-sumo**
…VMDR Inventory to CSE

PR provides guidance on forwarding Qualys VMDR Inventory to CSE. Currently only assetInventory is natively supported for _siemForward as other logTypes do not have parsers/mappers

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md [hunk 1]
```diff
@@ -50,14 +50,17 @@ To configure a Qualys VMDR Source:
...
 11. (Optional) **Collect KnowledgeBase Information**. This option is only available if you choose to collect vulnerability data. If selected, it will automatically download the vulnerability details from the Qualys KnowledgeBase for vulnerabilities detected within your environment.
 12. **Collect asset inventory**. This option consumes asset data from Qualys Global IT Asset Inventory API. The inventory data collected here will also be used in Cloud SIEM as inventory data. We recommend leaving the polling interval at the default 24 hours.
-13. When you are finished configuring the Source, click **Submit**.
+	 * To forward Qualys VMDR assetInventory to CSE, it is recommended to create a [Field Extraction Rule](https://help.sumologic.com/docs/manage/field-extractions/create-field-extraction-rule/) with the following criteria:
+		 * Scope: _sourceCategory="Enter Your Source Category Here from Step 5" "assetInventory"
+	   * Parse Expression: "true" as _siemForward
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md:62]: @mvirga-sumo , I've applied the code font where it looks like it's needed. Can you please take a look?

```suggestion
   * To forward Qualys VMDR assetInventory to CSE, it is recommended to create a [Field Extraction Rule](https://help.sumologic.com/docs/manage/field-extractions/create-field-extraction-rule/) with the following criteria:
      * Scope: `_sourceCategory="<enter your source category here from Step 5>" "assetInventory"`
      * Parse Expression: `true` as `_siemForward`
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md [hunk 2]
```diff
@@ -50,14 +50,17 @@ To configure a Qualys VMDR Source:
...
 6. (Optional) **Fields**. Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
    * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
-   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+	 * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md:53]:
```suggestion
   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md (hunk 1 above)
Comment (mvirga-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md:62]: @kimsauce Thank you for reviewing the PR. The double quotes on line 62 are required. If it's possible to put _sourceCategory="Enter Your Source Category Here from Step 5" "assetInventory" in a code block as well as "true" as _siemForward in a code block that would be excellent! 

Thank you!

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md:62]: @mvirga-sumo like this?

* Scope:
    ```
    _sourceCategory="<enter your source category here from Step 5>" "assetInventory"
   ```
* Parse Expression:
   ```
   "true" as _siemForward
   ```

Comment (mvirga-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/qualys-vmdr-source.md:62]: @kimsauce Yep that is perfect. Thank you!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**